ci/unit_tests/functions_deploy/main_test.py

```diff
@@ -109,6 +109,7 @@ def test_functionsUploadFromDirectory(self, useApikey):
             functionRespJson = functionResp.json()
             assert "Hello unit test!" in functionRespJson['greeting']
 
+    @pytest.mark.skipif(os.environ.get('TRAVIS_EVENT_TYPE') != "cron", reason="This test is nightly build only.")
     def test_pythonVersionFunctions(self):
```

Comment:
Maybe we can add `@pytest.mark.parametrize('useApikey', [True, False])` to this test and to `test_functionsInZip`

Reply:
Added.